file-service-api/src/main/java/uk/gov/justice/services/fileservice/domain/FileReference.java

```diff
@@ -17,11 +17,17 @@
     private final UUID fileId;
     private final JsonObject metadata;
     private final InputStream contentStream;
+    private final Boolean deleted;
```

Comment:
I prefer Booleans as objects because they can be null if unassigned. boolean would default to false which is harder to track down if there is a problem